tests/Predis/Command/Redis/CONFIG_Test.php

```diff
@@ -143,7 +143,7 @@ public function testReturnsTrueOnSuccessfulConfiguration(): void
     public function testThrowsExceptionWhenSettingUnknownConfiguration(): void
     {
         $this->expectException('Predis\Response\ServerException');
-        $this->expectExceptionMessage('ERR Unsupported CONFIG parameter: foo');
+        $this->expectExceptionMessageMatches("/ERR|CONFIG|foo|arguments|parameter/");
```

Comment:
Should we maybe have two full sentences here? That will make it easier to read.

Reply:
The sentence is different between 7 and anything lower so instead I was aiming to check that the key words are there which will work on all versions.

If you prefer to have a full sentence, I can do a small version check and test its content based on redis version

Reply:
Yeah, I think `(old error message|new error msg)` will be less difficult to read.

Reply:
Even better!